File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/team/lobby/ArenaLobbyCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.arena.team.lobby;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class ArenaLobbyCreatorHub extends CreatorHub {
    public ArenaLobby createLobby(String id, AbstractDataProvider dataProvider) throws Exception {
        return (ArenaLobby) this.create(id, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/team/lobby/ArenaLobbyCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.arena.team.lobby

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class ArenaLobbyCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): ArenaLobby {
        return super.create(itemID, dataProvider) as ArenaLobby
    }
}